data/tutorials/lg_01_functional_programming.md

```diff
@@ -168,7 +168,7 @@ val f : int -> int = <fun>
 # f 99;;
 - : int = 101
 ```
-In engineering this is sufficient [proof by example](humor_proof.html)
+In engineering this is sufficient [proof by example](humor_proof)
```

Comment:
We don't have the equivalent of humor_proof in v3.

```suggestion
In engineering, this is sufficient proof by example
```

Reply:
@tmattio will it make a comeback?